src/main/webapp/vulnerability/index.jsp

```diff
@@ -33,17 +33,64 @@
             <div id="vulndata"></div>
 
             <h3 style="margin-top:50px;">Affected Projects</h3>
-            <table id="vulnerableProjectsTable" class="table table-striped" data-toggle="table"
-                   data-url="<c:url value="/api/v1/vulnerability/source/${e:forUriComponent(param.source)}/vuln/${e:forUriComponent(param.vulnId)}/projects"/>"
-                   data-show-refresh="true" data-show-columns="true" data-search="true"
-                   data-response-handler="formatProjectsTable" data-height="100%">
-                <thead>
-                <tr>
-                    <th data-align="left" data-field="projecthref">Name</th>
-                    <th data-align="left" data-field="version">Version</th>
-                </tr>
-                </thead>
-            </table>
+            <div class="panel with-nav-tabs panel-default tight">
```

Comment:
Lets make this into a single table again, but with the additional Component, Comment, and Analysis State fields with a table toolbar containing a checkbox to optionally include suppressed. When suppressed is included, the Suppressed field would be made visible in the table, the URL would be updated to specify `?includeSuppressed=true` and the table would silently refresh.

Reply:
If you need my help here. Not a problem. Just let me know.

Reply:
Ack.. Due to time constraint, I couldn't able to finish this on time. If I suppose to do, it will take bit time since I have to look at a single-table structure. Therefore IMHO, It would be great if you help on this.